### PR TITLE
Changes Related to WSO2 CARBON PATCH 5.0.0 0286

### PR DESCRIPTION
**Pull request by rashm1n**
## Purpose

Fixes wso2/product-apim#11152

### CODE REVIEW

File: features/org.wso2.analytics.apim.feature/src/main/resources/siddhi-files/apim_alert_frequent_tier_limit_hitting_0.siddhi [hunk 1]
```diff
@@ -75,15 +77,15 @@ insert into TemporaryUserStream;
 
 --this query ensures that the same type of alert is not generated for same apiCreator again for a time defined in #window.time(),
 @info(name = 'Query contains Suppression logic')
-from TemporarySubscriberStream#window.length(1) as a left outer join SuppressedTemporarySubscriberStream#window.time(10 minute) as b
+from TemporarySubscriberStream#window.length(1) as a left outer join SuppressedTemporarySubscriberStream#window.time(10  minutes) as b
```
Comment (ruks) [inline features/org.wso2.analytics.apim.feature/src/main/resources/siddhi-files/apim_alert_frequent_tier_limit_hitting_0.siddhi:80]: do we need additional space before the `minutes`?

File: features/org.wso2.analytics.apim.feature/src/main/resources/siddhi-files/apim_alert_frequent_tier_limit_hitting_0.siddhi [hunk 2]
```diff
@@ -75,15 +77,15 @@ insert into TemporaryUserStream;
...
  	on (a.apiCreator == b.apiCreator and a.apiVersion == b.apiVersion and a.applicationId == b.applicationId)
 select a.apiCreator, a.apiName, a.apiVersion, a.applicationId, a.applicationName, a.apiCreatorTenantDomain, a.userTenantDomain, a.subscriber, a.numHits
  	having b.apiCreator is null
 insert into SuppressedTemporarySubscriberStream ;
 
 --this query ensures that the same type of alert is not generated for same user again for a time defined in #window.time()
 @info(name = 'Alert Suppression')
-from TemporaryUserStream#window.length(1) as a left outer join SuppressedTemporaryUserStream#window.time(10 minute) as b
+from TemporaryUserStream#window.length(1) as a left outer join SuppressedTemporaryUserStream#window.time(10  minutes) as b
```
Comment (ruks) [inline features/org.wso2.analytics.apim.feature/src/main/resources/siddhi-files/apim_alert_frequent_tier_limit_hitting_0.siddhi:88]: do we need additional space before the `minutes`?

**Comment by rashm1n** [comment]
@ruks the changes are made

**Comment by rashm1n** [comment]
The changes are already applied to the carbon-apimgt.

**Pull request closed without merging**